packages/react-reconciler/src/__tests__/ReactTransitionTracing-test.js

```diff
@@ -477,4 +477,365 @@ describe('ReactInteractionTracing', () => {
+          `onMarkerComplete(${transitioName}, ${markerName}, ${startTime}, ${endTime})`,
+        );
+      },
+    };
+
+    let navigateToPageTwo;
+    function App() {
+      const [navigate, setNavigate] = useState(false);
+      navigateToPageTwo = () => {
+        setNavigate(true);
+      };
+
+      return (
+        <div>
+          {navigate ? (
+            <React.unstable_TracingMarker name="outer marker">
+              <Suspense fallback={<Text text="Outer..." />}>
+                <AsyncText text="Outer Text" />
+                <Suspense fallback={<Text text="Inner One..." />}>
+                  <React.unstable_TracingMarker name="marker one">
+                    <AsyncText text="Inner Text One" />
+                  </React.unstable_TracingMarker>
+                </Suspense>
+                <Suspense fallback={<Text text="Inner Two..." />}>
+                  <React.unstable_TracingMarker name="marker two">
+                    <AsyncText text="Inner Text Two" />
+                  </React.unstable_TracingMarker>
+                </Suspense>
+              </Suspense>
+            </React.unstable_TracingMarker>
+          ) : (
+            <Text text="Page One" />
+          )}
+        </div>
+      );
+    }
+
+    const root = ReactNoop.createRoot({transitionCallbacks});
+    await act(async () => {
+      root.render(<App />);
+      ReactNoop.expire(1000);
+      await advanceTimers(1000);
+
+      expect(Scheduler).toFlushAndYield(['Page One']);
+    });
+
+    await act(async () => {
+      startTransition(() => navigateToPageTwo(), {name: 'page transition'});
+
+      ReactNoop.expire(1000);
+      await advanceTimers(1000);
+
+      expect(Scheduler).toFlushAndYield([
+        'Suspend [Outer Text]',
+        'Suspend [Inner Text One]',
+        'Inner One...',
+        'Suspend [Inner Text Two]',
+        'Inner Two...',
+        'Outer...',
+        'onTransitionStart(page transition, 1000)',
+      ]);
+
+      ReactNoop.expire(1000);
+      await advanceTimers(1000);
+      await resolveText('Inner Text Two');
+      expect(Scheduler).toFlushAndYield([]);
+
+      ReactNoop.expire(1000);
+      await advanceTimers(1000);
+      await resolveText('Outer Text');
+      expect(Scheduler).toFlushAndYield([
+        'Outer Text',
+        'Suspend [Inner Text One]',
+        'Inner One...',
+        'Inner Text Two',
+        'onMarkerComplete(page transition, marker two, 1000, 4000)',
+      ]);
+
+      ReactNoop.expire(1000);
+      await advanceTimers(1000);
+      await resolveText('Inner Text One');
+      expect(Scheduler).toFlushAndYield([
+        'Inner Text One',
+        'onMarkerComplete(page transition, marker one, 1000, 5000)',
+        'onMarkerComplete(page transition, outer marker, 1000, 5000)',
+        'onTransitionComplete(page transition, 1000, 5000)',
+      ]);
+    });
+  });
+
+  // @gate enableTransitionTracing
+  it.skip('marker interaction cancelled when name changes', async () => {
```

Comment:
Is this one meant to be skipped?

Reply:
Yes! Skipping because we need to talk about what to do when the marker's name changes. Leaving this test here for now as a reminder (and also cause I wrote it already)